platform-server/src/main/resources/application-local.yml

```diff
@@ -0,0 +1,5 @@
+spring:
```

Comment:
stuff that relates to the local profile should not be commited, please add `application-local.yml` to gitignore